manifests/params.pp

```diff
@@ -329,10 +329,10 @@
   case $::osfamily {
     'Debian' : {
       $agent_restart_command = "/usr/sbin/service ${service_name} reload"
-      if (
-        ($::operatingsystem == 'Debian') and (versioncmp($::operatingsystemrelease, '8.0') >= 0) or
-        ($::operatingsystem == 'Ubuntu') and (versioncmp($::operatingsystemrelease, '15.04') >= 0)
-      ) {
+      if
+      $::operatingsystem == 'Debian' or
```

Comment:
Indenting looks a bit weird here. Would it be good to keep parenthesis around the entire statement? `if (..) {`

Reply:
Done. I did put the whole condition into one long line as it would look more strange because of the formatting rules we have in place otherwise.